src/TourManager/Repository/Entities/Booking.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Travely.TourManager.Repository.Repositories.Entities
+{
+    public class Booking
```

Comment:
right now I'm a little bit confused regarding the model naming. I'd name the repo entities with some name like BookingEntity, and model name would be Booking, which is more practical. Naming it BookinModel, or BookingData creates artificial misunderstanding tho. If possible please change 🤔